src/lib.rs

```diff
@@ -62,6 +63,17 @@ impl Config {
             assets,
         })
     }
+
+    /// Returns the directory where output files should be placed
+    pub fn output_path(&self) -> PathBuf {
+        self.manifest_path.join("target").join("doc")
+    }
+
+    /// Open the generated docs in a web browser.
+    pub fn open_docs(&self) -> Result<()> {
+        open::that(self.output_path().join("index.html")).unwrap(); // TODO: Error handling
```

Comment:
I *think* because we use error-chain, you just add in the wrapper for its type and it should Just Work? I'm still learning error-chain